.goreleaser.yml

```diff
@@ -9,105 +9,62 @@ env:
 
 # Builds
 builds:
-  # GoShimmer AMD64 all OS
-  - id: goshimmer
-    binary: goshimmer
-    flags:
-      - -tags=rocksdb
-    ldflags:
-      - -s -w -X github.com/iotaledger/goshimmer/plugins/banner.AppVersion={{.Version}}
-    goos:
-      - linux
-    goarch:
-      - amd64
```

Comment:
We don't build GoShimmer?

Reply:
Correct. We should focus on 1 setup (Docker).